src/applications/vaos/api/sites-supporting-var.json

```diff
@@ -1,619 +1,1004 @@
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "631",
+      "type": "object_type",
+      "attributes": {
+        "name": "Leeds",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "650",
+      "type": "object_type",
+      "attributes": {
+        "name": "Providence",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "508",
+      "type": "object_type",
+      "attributes": {
+        "name": "Decatur",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "509",
+      "type": "object_type",
+      "attributes": {
+        "name": "Augusta",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "521",
+      "type": "object_type",
+      "attributes": {
+        "name": "Birmingham",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "534",
+      "type": "object_type",
+      "attributes": {
+        "name": "Charleston",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "544",
+      "type": "object_type",
+      "attributes": {
+        "name": "Columbia",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "557",
+      "type": "object_type",
+      "attributes": {
+        "name": "Dublin",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "619",
+      "type": "object_type",
+      "attributes": {
+        "name": "Central Alabama",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "679",
+      "type": "object_type",
+      "attributes": {
+        "name": "Tuscaloosa",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "504",
+      "type": "object_type",
+      "attributes": {
+        "name": "Amarillo",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "519",
+      "type": "object_type",
+      "attributes": {
+        "name": "Big Spring",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "549",
+      "type": "object_type",
+      "attributes": {
+        "name": "North Texas",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "671",
+      "type": "object_type",
+      "attributes": {
+        "name": "South Texas",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "674",
+      "type": "object_type",
+      "attributes": {
+        "name": "Central Texas",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "740",
+      "type": "object_type",
+      "attributes": {
+        "name": "Harlingen",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "756",
+      "type": "object_type",
+      "attributes": {
+        "name": "El Paso",
+        "timezone": "US/Central"
+      }
+    },
+    {
+      "id": "541",
+      "type": "object_type",
+      "attributes": {
+        "name": "Cleveland",
+        "timezone": "US/Eastern"
+      }
+    },
+    {
+      "id": "983",
```

Comment:
I think I would cut this file down to just these two: 983 and 984. The rest isn't that useful as mock data, I don't think.